.vscode/tasks.json

```diff
@@ -14,6 +14,20 @@
         "panel": "new"
       },
       "problemMatcher": []
+    },
+    {
+      "label": "Rebuild and Start Addon",
+      "type": "shell",
+      "command": "docker exec -it hassio_cli sh -c \"ha addons rebuild local_${input:addonName} && ha addons start local_${input:addonName}\" && sudo docker logs --follow addon_local_${input:addonName}",
```

Comment:
The input needs to be defined before it can be used <https://code.visualstudio.com/docs/editor/variables-reference#_input-variables>

Reply:
Make sense, @davet2001 Is that something you can do so we can bring this PR home?

../Frenck

Reply:
Sure, will do.